freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/user/model/CreateUsersRequest.java

```diff
@@ -0,0 +1,39 @@
+package com.sequenceiq.freeipa.api.v1.freeipa.user.model;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.sequenceiq.freeipa.api.v1.freeipa.stack.doc.FreeIpaModelDescriptions;
+import com.sequenceiq.freeipa.api.v1.freeipa.user.doc.UserModelDescriptions;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel("CreateUsersV1Request")
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CreateUsersRequest {
+    @NotNull
+    @ApiModelProperty(value = FreeIpaModelDescriptions.ENVIRONMENT_ID, required = true)
+    private String environmentCrn;
```

Comment:
I think we should call this environmentId to make it to similar to existing post request

Reply:
done!